app/subjects/index.cjsx

```diff
@@ -0,0 +1,81 @@
+React = require 'react'
+apiClient = require '../api/client'
+authClient = require '../api/auth'
+talkClient = require '../api/talk'
+getSubjectLocation = require '../lib/get-subject-location'
+FavoritesButton = require '../collections/favorites-button'
+ChangeListener = require '../components/change-listener'
+PromiseRenderer = require '../components/promise-renderer'
+NewDiscussionForm = require '../talk/discussion-new-form'
+{Navigation} = require 'react-router'
+
+module?.exports = React.createClass
+  displayName: 'Subject'
+  mixins: [Navigation]
+
+  getInitialState: ->
+    subject: null
+
+  componentWillMount: ->
+    @setSubject()
```

Comment:
Ok just added that in https://github.com/zooniverse/Panoptes-Front-End/commit/b4f502a2afd711e714151da13d51a31e90c008e1